pkg/core/info/enemy.go

```diff
@@ -4,6 +4,21 @@ import (
 	"github.com/genshinsim/gcsim/pkg/core/attributes"
 )
 
+// TODO: protobuf?
```

Comment:
something to consider when integrating into pipeline / creating issue for this integration (can be done in later PR)